frontend/src/modules/controllers/ValueModal.tsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * @summary Re-usable wrapper components to display & control value
```

Comment:
Where would these components be used? Only in the Set Value modal, or also elsewhere (e.g. on the quickstart page)?

Reply:
At this moment its not in use. We have modified it to suit to Multi Step popup and named it as `SetValueContent` component